_posts/2015-12-11-License-and-You.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: 'Licenses for OSS Code'
+date: 2015-12-10T00:00:00.000Z
+comments: false
+categories: [ios, mobile, review, video, oss]
+author: orta
+---
+
+As a part of our internal Lunch and Learn series, I  gave a talk to our developers with an overview of the different types of source code licenses available.
+
+We always recommend MIT within Artsy, but understanding what the other ones are and why we made the choices we have done is valuable.
```

Comment:
I could have sworn we had a blog post discussing how we manage OSS repos (under the dev's account, etc) but I can't find it...